doc/source/visualizing/plots.rst

```diff
@@ -1099,6 +1099,83 @@ function of radius. The xlabel is set to "Radius", for all plots, and the ylabel
    plot.set_ylabel("x-velocity", "velocity in x direction")
    plot.save()
 
+Adding plot title
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Plot title can be set via the
+:meth:`~yt.visualization.profile_plotter.ProfilePlot.annotate_title` function.
+It accepts a string argument which is the plot title and an optional ``field`` parameter which specifies
+the field for which plot title should be added. ``field`` could be a string or a list of string.
+If ``field`` is not passed, plot title will be added for the fields.
+
+In the following example we create a plot and set the plot title.
+
+.. python-script::
+
+   import yt
+   ds = yt.load("enzo_tiny_cosmology/DD0046/DD0046")
+   ad = ds.all_data()
+   plot = yt.ProfilePlot(ad, "density", ["temperature"], weight_field=None)
+   plot.annotate_title("Temperature vs Density Plot")
+   plot.save()
+
+Another example where we create plots from profile. By specifying the fields we can add plot title to a
+specific plot.
+
+.. python-script::
+
+   import yt
+   ds = yt.load('enzo_tiny_cosmology/DD0046/DD0046')
+   sphere = ds.sphere("max", (1.0, "Mpc"))
+   profiles = []
+   profiles.append(yt.create_profile(sphere, ["radius"], fields=["density"],n_bins=64))
+   profiles.append(yt.create_profile(sphere, ["radius"], fields=["dark_matter_density"],n_bins=64))
+   plot = yt.ProfilePlot.from_profiles(profiles)
+   plot.annotate_title("Plot Title: Density", "density")
+   plot.annotate_title("Plot Title: Dark Matter Density", "dark_matter_density")
+   plot.save()
+
+Here, ``plot.annotate_title("Plot Title: Density", "density")`` will only set the plot title for the ``"density"``
+field. Thus, allowing us the option to have different plot titles for different fields.
+
+
+Annotating plot with text
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
same here

Reply:
Thank you for pointing out! I fixed those now.